### DISCUSSION TITLE
[feature request] navigation link

### DISCUSSION THREAD

**Discussion by uvesco** [opengisch/QField#5264] [Ideas]
It would be very useful in the field to have a new feature: a link to a coordinate to be opened in a navigator app (e.g. google maps). I'm always looking for the same place to navigate to when I have to move to a different sector by car.

**Comment by ZsanettMed** [comment]
Hello @uvesco!

Thanks for sharing your idea! It's now on our [QField Idea Platform ](https://ideas.qfield.org/app-feature-requests/p/linkt-to-navigation)where you and others can vote on it. This really helps us decide what to work on next. If you're interested in funding the feature to speed things up, please email us at [sales@qfield.org](mailto:sales@qfield.org).

p.s.: Since we are moving all feature requests to the new platform, I will close this ticket.

Best,
Zsanett
